.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust PR CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Does this work on `macos-latest` and `windows-latest`? Can we add those if it does?

Reply:
good looks